fastlane/Fastfile

```diff
@@ -603,8 +605,17 @@ platform :android do
     update_identifiers
     replace_assets
     link_sentry_android
-    build_android
-    move_apk_to_root
+
+    build_type = ENV['ANDROID_BUILD_TYPE'].nil? || ENV['ANDROID_BUILD_TYPE'].empty? ? 'both' : ENV['ANDROID_BUILD_TYPE']
```

Comment:
I like the idea of building it both as an optional step, but I would leave the default as the apk only so that it does not change the current workflow. We can have `ANDROID_BUILD_TYPE` explicitly set to `both`.

In fact I would also have it work like this:
1. Call the env var `ANDROID_BUILD_TASK` as build_type really refers to either Debug or Release
2. instead of setting the env var as `apk`, `aab` or `both` you can set it as an array like `['assemble']`, `['bundle']` or `['assemble', 'bundle']
3. combine `move_apk_to_root` and `move_aab_to_root` as one function the same for `build_android_apk` and `build_android_bundle`, perhaps rename to `move_android_to_root` and `build_android` and read from the env var to determine the files that needs to be build and moved
4. Always default to current workflow if env var is not set

Perhaps this approach will reduce the amount of code duplication.